app/pages/lab/data-exports.jsx

```diff
@@ -72,15 +72,13 @@ export default function DataExports (props) {
             />
           </div>
           <div className="row">
-            <DataExportButton
-              project={props.project}
-              buttonKey="projectDetails.workflowContentsExport"
-              exportType="workflow_contents_export"
-            />
+            <p>
+              <strong>Looking for workflow contents exports?</strong> The workflow contents exports have been merged into the normal workflow export. The "strings" column is now available directly in the workflows export, and the "version" column from the workflow contents export is called "minor_version" in the workflows export. This means you no longer need to look up rows from two files in order to know what the actual setup of the workflow was for the version number specified by a classificatio.
```

Comment:
@mrniaboc think this is clear enough to communicate that the button's gone away but the data is just moved to another csv?

Reply:
@marten There's an "n" missing from classification at the end.